dev-requirements.txt

```diff
@@ -1,3 +1,5 @@
-protobuf==3.5.1
+ed25519==1.5
```

Comment:
https://github.com/warner/python-ed25519#not-recommended-for-new-applications-use-pynacl-instead

>For new applications, I recommend you use pynacl instead of this repository. PyNaCl is larger and takes longer to build (it contains the complete NaCl/libsodium library, not just the ed25519 portion), but it is well-maintained by the diligent and conscientious PyCA team, whereas I've allowed this repository to languish. PyNaCl is also about 10-20 times faster. A guide for migration fron python-ed25519 to PyNaCl is included below.